docs/UserGuide.md

```diff
@@ -98,7 +109,7 @@ To get started,
 --------------------------------------------------------------------------------------------------------------------
 ## <i class="fa-solid fa-bolt-lightning"></i> Parameter shortcuts
 
-All parameters in KonTActs come equipped with their equivalent shortcuts.
+Most parameters in KonTActs come equipped with their equivalent shortcuts.
```

Comment:
Shouldn't this be all parameters since all the prefixes we have support the shortcuts?

Reply:
Right thats true